packages/bot-web-ui/src/stores/server-side-bot-store.ts

```diff
@@ -661,4 +669,27 @@ export default class ServerBotStore {
     setSelectedStrategy = (strategy: string) => {
         this.selected_strategy = strategy;
     };
+
+    onCloseDialog = () => {
+        this.is_dialog_open = false;
+    };
+
+    showClearStatDialog = () => {
+        this.onCancelButtonClick = this.onCloseDialog;
+        this.onOkButtonClick = () => {
+            this.resetJournal();
+            this.resetTransactions();
+            this.onCloseDialog();
+        };
+        this.onCloseDialog();
+        this.dialog_options = {
+            title: localize('Are you sure?'),
+            message: localize(
+                'This will clear all data in the summary and journal panels. All counters will be reset to zero.'
+            ),
+            cancel_button_text: localize('Cancel'),
+            ok_button_text: localize('Ok'),
+        };
+        this.is_dialog_open = true;
+    };
```

Comment:
@maryia-matskevich-deriv can we not add this in the store?
I think we can handle this on the component level.

we do have reset methods that call be called directly from the components.

Reply:
@sandeep-deriv we can do it using a component, but I decided to use the same approach as we have in the run panel store because I see a benefit. If we need to show again some modal window with action buttons then we don't need to pass a component, we need only create one more method like this one. 
If this is not a sufficient advantage of this approach, I will replace it with a component transfer.👌

Reply:
The current approach can reuse the same component in different places

Reply:
I think we can reuse it form the component level as well. @maryia-matskevich-deriv 

If we were reusing it with different stores than it makes sense but even in that case only the texts should be sent dynamically. But in current case we already added the component in the `server-side-bot.tsx` file. 

But here we are just putting more things in the store when this can be handled in components.

Reply:
> @sandeep-deriv we can do it using a component, but I decided to use the same approach as we have in the run panel store because I see a benefit. If we need to show again some modal window with action buttons then we don't need to pass a component, we need only create one more method like this one. If this is not a sufficient advantage of this approach, I will replace it with a component transfer.👌

1. The run panel store is always available, but the server bot will be conditionally rendered.
2. We need to move out all these dialogs and static implementations from the stores to components down the line.

Reply:
The current approach is limited to functioning with a single store, similar to the setup we have in the run panel store. Introducing a new function within the store will automatically trigger one component in the `server-side-bot.tsx`. It's crucial to understand that we need to supply not just the text, but also the methods, which can be passed directly within the store. This highlights the importance of a more versatile and scalable solution to accommodate future growth and complexity.
I'm not insisting on my decision🙂

Reply:
ok, let me change it👌

Reply:
changed✅